pkg/template/templateprocessing/template.go

```diff
@@ -143,6 +144,17 @@ func DeprecatedGetParameterByNameInternal(t *templateapi.Template, name string)
 	return nil
 }
 
+// DeprecatedGetParameterByNameInternalV1 searches for a Parameter in the Template
+// based on its name.
+func DeprecatedGetParameterByNameInternalV1(t *templatev1.Template, name string) *templatev1.Parameter {
```

Comment:
If this is deprecated, what's the new way, and why you're not using it, instead of copying the deprecated method.